src/contexts/instance-context.js

```diff
@@ -10,13 +10,21 @@ export const InstanceProvider = ({ children }) => {
     const { helxAppstoreUrl } = useEnvironment();
     const { addActivity, updateActivity } = useActivity();
     const [openedTabs, setTabs] = useState([]);
+    const [pollingTimerIDs, setpollingTimerIDs] = useState([])
```

Comment:
teeny thing, but i think you wanted a capital "P" in setpollingTimerIDs

Reply:
Nice catch on this! Right, definitely should be a capital 'P' there, will change that. Thanks.